requirements/test.txt

```diff
@@ -30,6 +30,7 @@ future==0.17.1            # via pyjwkest
 httmock==1.2.6
 httpretty==0.9.6
 idna==2.7                 # via requests
+jsonfield==2.0.2
```

Comment:
@MichaelRoytman I think with @nedbat's change, you don't need to put jsonfield in the test/doc/etc requirements files. Just the base file will do the trick.

Reply:
@MichaelRoytman You should edit just one .in file to add the requirement (base.in).  Then "make upgrade" will update all the .txt files.